server/src/main/java/org/eclipse/lsp/cobol/service/CFASTBuilder.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.service;
+
+import org.eclipse.lsp.cobol.core.model.extendedapi.ExtendedApiResult;
+import org.eclipse.lsp.cobol.core.model.tree.Node;
+
+/**
+ * CF AST builder interface.
+ */
+public interface CFASTBuilder {
```

Comment:
Builder reflects more class name than Interface name

Reply:
Could you please clarify it, I'm not sure that I understand the issue?

Reply:
> Could you please clarify it, I'm not sure that I understand the issue?

Just name of interface.  CFASTBuilder (e.g. CarBuilder) sounds more class than interface.  We may use "Builder" or "Buildable" if not suits we can leave it as it is  ;)

Reply:
@temanbrcom is it necessary to have an interface at all? Can it be just service class?

Reply:
I would prefer to have an interface - it may be mocked for testsing